mods/common/chrome/ingame-infostats.yaml

```diff
@@ -103,11 +103,11 @@ Container@SKIRMISH_STATS:
 							Template: ANONYMOUS_PLAYER_TOOLTIP
 						Label@NAME:
 							X: 29
-							Width: 191
+							Width: 150
```

Comment:
Revert every yaml change that isn't adding the kick button. #20100 already did all the changes for you